packages/dev/parcel-packager-docs/DocsPackager.js

```diff
@@ -291,8 +292,15 @@ function mergeInterface(obj) {
         console.log('ext should not be null', obj);
         continue;
       }
-      merge(properties, mergeInterface(ext).properties);
+      let merged = mergeInterface(ext);
```

Comment:
slight difference to what we did here https://github.com/adobe/react-spectrum/pull/3997/files#diff-ea18fe8150ebc7897b4f4ec1e895cbd1fe6863f1a9f614d7d4d6addc117a6ba8R310

any reason?

Reply:
I didn't know about that one. Wouldn't this one handle when an interface extends something that isn't an interface, but that other one wouldn't?

Reply:
haha, you helped @LFDanLu and I write those lines ;P
and yeah, looks like that is the difference

Reply:
🙈